### DISCUSSION TITLE
Edit text with a bluetooth keyboard on kobo sage

### DISCUSSION THREAD

**Discussion by swagtoast** [koreader/koreader#12232] [Ideas]
Hi, thank you for your work. I love Koreader and - as a writer - the Koreader text editor is my work companion, as I can use my ereader as a typewriter.
I'm currently using a Kobo Sage with an external bluetooth keyboard. As Koreader doesn't support bluetooth, I bought a usb-c splitter dongle to connect my keyboard both to the kobo and to the power supply (as Kobo otg doesn't supply any power); but those dongles lack in quality, as i changed them 2 times in the last week: they only work for a few days.
I was wondering if it is possible to implement bluetooth connection: it would be a killer feature for me an many others in my situation, as we could straight connect the external keyboard without any cable and dongle, and start writing.
Unfortunally, I don't have any coding skill, but would love to help.
Thank you again for your precious work.

**Comment by hius07** [comment]
The main thread on bluetooth: https://github.com/koreader/koreader/issues/9059.

**Comment by NiLuJe** [comment]
Stupid question is stupid: what's preventing you from using a USB keyboard via OTG instead?
